src/shx.js

```diff
@@ -37,10 +37,13 @@ export const shx = (argv) => {
     newArgs = [];
     let lookingForSubstString = true;
     args.forEach((arg) => {
-      const match = arg.match(/^s\/(.*)\/(.*)\/(g?)$/);
+      const match = arg.match(/^s\/(.*[^\\])\/(.*[^\\])\/(g?)$/);
```

Comment:
@ariporad A sed-regex is allowed to have backslashes in it, but it's not allowed to end in a backslash. The end of a sed-regex is designated by the first appearance of a non-escaped `/`. If you want to include a `/` in the regex, you would use `\/`. I think your JS-regex would prevent that from working.